pkgs/leak_tracker/lib/src/leak_tracking/_object_record.dart

```diff
@@ -48,10 +48,10 @@ class ObjectRecord {
   int? _disposalGcCount;
 
   void setDisposed(int gcTime, DateTime time) {
-    // TODO(polina-c): handle double disposal in a better way
-    // https://github.com/dart-lang/leak_tracker/issues/118
-    // Noop if object is already disposed.
-    if (_disposalGcCount != null) return;
+    if (_disposalGcCount != null) {
+      // It is not responsibility of leak tracker to check for double disposal.
```

Comment:
More of a CL description would help clarify what this CL is doing. Is it handling tracking or just ignoring the double dispose problem? Would expect issue/118 would be closed if the design decision is to not handle the case.

Reply:
And added description.